docs/08-tips-and-tricks.md

```diff
@@ -0,0 +1,121 @@
+# Tips and tricks
+This section explains some tips and tricks to familiarize with OpenShift 4.
+
+## Ignition files
+
+You can use `jq` to read the files easily:
+
```

Comment:
you don't describe where the files come from and there's no real discussion about ignition in general so this feels very out of left field

Reply:
Added